cluster/saltbase/salt/kube-controller-manager/kube-controller-manager.manifest

```diff
@@ -137,6 +138,7 @@
         { "name": "srvkube",
         "mountPath": "{{srv_kube_path}}",
         "readOnly": true},
+        {{flexvolume_hostpath_mount}}
```

Comment:
Does this apply to all kube-controller-manager deployments? Or just GCE? Also on GKE can users even deploy a Flex driver to a master machine?

Reply:
It applies to all deployments. No Flex drivers can't be deployed on GKE masters, so the HostPath will exist but won't be used.